fastlane/BaseFastfile

```diff
@@ -203,6 +203,23 @@ platform :ios do
     frameit(white: true)
   end
 
+  desc "Takes localization snapshots \
+  :devices - a list of devices to take snapshots on \
+  :languages - a list of languages to take snapshots of \
+  :output_directory - the directory to output the snapshots to"
+  lane :snapshotL10n do |options|
+    devices = options[:devices].split(",") || ["iPhone 6"]
```

Comment:
Gave it another try this morning with a fresh repo clone and I'm getting the following error when providing no arguments:

`undefined method `split' for nil:NilClass`

I think we need to assign options[:devices]/options[:languages] the defaults prior to splitting.